db_model/orm_model.py

```diff
@@ -116,6 +116,7 @@ class BuildAction(Base):
     run_id = Column(Integer,
                     ForeignKey('runs.id', deferrable=True, initially="DEFERRED",
                                ondelete='CASCADE'))
+    # hash of the build command
```

Comment:
If I change the just the column name this way `build_cmd = Column("build_cmd_hash", String)
` is not enough.
I got these error messages:
`[ERROR] - (psycopg2.ProgrammingError) column build_actions.build_cmd_hash does not exist`

The proper way to do it is to create an alembic schema upgrade script, which will rename the column.